addons/scratchstats/userscript.js

```diff
@@ -0,0 +1,115 @@
+function createItem(number, label) {
+  const item = document.createElement("div");
+  const numberDiv = document.createElement("div");
+  item.appendChild(numberDiv);
+  numberDiv.className = "sa-stats-number";
+  numberDiv.innerText = number;
+  const labelDiv = document.createElement("div");
+  item.appendChild(labelDiv);
+  labelDiv.innerText = label;
+  return item;
+}
```

Comment:
i think this is supposed to go inside the `export default async function ({ addon, msg, console }) {`

Reply:
it doesn't have to

Reply:
it doesn’t? I always thought it did.

Reply:
i dont think it needs to be part of the exported module

Reply:
> i dont think it needs to be part of the exported module

It doesn't. Exported functions can always use functions that are not exported.

Reply:
@RedGuy12 The function that is exported by default is simply a function that runs when everything is ready, similarly to `int main() {}` in other programming languages. Only the default function gets references to `addon`, `msg` and other objects, but you can always pass those as arguments for other functions as well.